src/schema/v2/artwork/artworkCompletenessChecklistItem.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  GraphQLBoolean,
+  GraphQLInt,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+import { ResolverContext } from "types/graphql"
+
+export const ArtworkCompletenessChecklistItemType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "ArtworkCompletenessChecklistItem",
+  fields: {
+    type: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "The type/identifier of the validation",
+      resolve: ({ type }) => type,
+    },
+    completed: {
+      type: new GraphQLNonNull(GraphQLBoolean),
+      description: "Whether this checklist item is completed",
+      resolve: ({ completed }) => completed,
+    },
+    weight: {
+      type: new GraphQLNonNull(GraphQLInt),
+      description: "The score weight of this validation",
+      resolve: ({ weight }) => weight,
+    },
```

Comment:
I would remove the weight for now because it's not needed in Volt. The weight is internal to the scoring algorithm and may not be relevant anymore when we modify it.

Reply:
Yeah, I just think we need to make it a bit more obvious to avoid confusion in the future.

Reply:
For sure, I removed it for now.